server/server.py

```diff
@@ -1,24 +1,18 @@
 # server.py
-from flask import Flask, render_template, request
+from flask import Flask, render_template, request, url_for
 import speak
 
-# text = "AI AI minha machadinha !!"
-# lang = "pt"
-
 app = Flask(__name__, static_folder="../static/dist", template_folder="../static")
 @app.route("/", methods=["GET","POST"])
 def index():
     return render_template("index.html")
 
-@app.route("/hello", methods=["GET","POST"])
-def hello():
-    text=''
-    lang=''
-    if request.method == "POST":
-        text = request.form["text"]
-        lang = request.form["lang"]
-        print(text)
-    return speak.get_speak(text,lang)
+@app.route("/hello/<text>/<lang>")
+def hello(text,lang):
+    if request.method == "GET":
+        speak.get_speak(text,lang)
+        return url_for("static", filename="mp3_path")
 
 if __name__ == "__main__":
-    app.run()
\ No newline at end of file
+    app.run()
+    
```

Comment:
The new line should be left completely empty "including white spaces"

![image](https://user-images.githubusercontent.com/26286907/79394987-5e6e0a00-7f81-11ea-9f69-350d1bbb4c32.png)